client/src/ui/molecules/PeoplePickerSelector.test.tsx

```diff
@@ -149,4 +219,23 @@ describe('PeoplePickerSelector', (): void => {
         fireEvent.click(baseElement.querySelector('.modal .button'));
         expect(doneMock).not.toHaveBeenCalled();
     });
+
+    it('should call the onSearch callback when the user types a string into the search box', async (): Promise<
+        void
+    > => {
+        const searchMock = jest.fn();
+        const { baseElement } = render(
+            <PeoplePickerSelector
+                initialySelected={[]}
+                people={people}
+                onDone={jest.fn()}
+                onSearch={searchMock}
+                label=" "
+                toggle={jest.fn()}
+                isShowing={true}
+            />,
+        );
+        await fireEvent.click(baseElement.querySelector('input'), { target: { value: 'someSearch' } });
+        expect(searchMock).toHaveBeenCalledTimes(1);
+    });
```

Comment:
is this working now?

Reply:
No, my bad didn't run the full test suite